src/header/header.scss

```diff
@@ -219,6 +219,10 @@
 
     > .iui-icon {
       flex-shrink: 0;
+
+      &:not(.iui-header-button-icon) {
```

Comment:
Are these changes really necessary for this PR?

Reply:
We could remove this if it will help it pass.  Without this, the icons do not align.  We also won't be able to show a photo thumbnail, we'll only have icons for now.

**With this CSS:**
![header-menu-with](https://user-images.githubusercontent.com/849817/124321867-672e0a00-db4c-11eb-9668-57ac17b64baa.png)

**Without this CSS:**
![header-menu-without](https://user-images.githubusercontent.com/849817/124321882-6d23eb00-db4c-11eb-95d5-9e25acbe2aa0.png)

Worth pointing out tho that [the initial request / bug filed](https://github.com/iTwin/iTwinUI/issues/85) is using the double line menu items from the context of the header button.

Reply:
Huh, how this affects alignment?

Reply:
Sorry, I must have misunderstood what the initial comment was questioning.

I've removed this CSS since we also removed `.iui-expanded` from the header button.